powerauth-java-server/pom.xml

```diff
@@ -51,6 +51,8 @@
         <dependency>
             <groupId>org.springframework.ws</groupId>
             <artifactId>spring-ws-core</artifactId>
+            <!-- TODO - update/remove version shortly before final release, 3.0.6.RELEASE is the current version which causes startup issues on Wildfly -->
```

Comment:
Is this a good idea, won't we really forget this? I would prefer downgrading the version and updating sooner before the release in case the issue is fixed.

Reply:
Let's keep the issue open then. I can test PA server on Wildfly in this issue branch, otherwise it would not start.

Reply:
The issue is fixed (I verified it), we just need to wait a bit for dependencies to line up with this `spring-ws-security` version.